ultralytics/utils/export.py

```diff
@@ -41,7 +41,8 @@ def export_onnx(
         do_constant_folding=True,  # WARNING: DNN inference with torch>=1.12 may require do_constant_folding=False
         input_names=input_names,
         output_names=output_names,
-        dynamic_axes=dynamic or None,
+        dynamic_shapes=dynamic or None,
```

Comment:
The dynamic shapes argument is a dictionary specified in https://pytorch.org/docs/stable/export.html#torch.export.export. We have made the `dynamo=True` mode compatible with `dynamic_axes` for simple cases. If `dynamic_axes` works you may continue using it

Reply:
Ok got it! I'll try to keep for now. I've added onnxscript as a missing dependency.